.github/workflows/start-sync-test.yml

```diff
@@ -19,9 +19,18 @@ jobs:
         env:
           GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
 
-      - uses: Chia-Network/actions/github/jwt@main
+      - name: Trigger sync test workflow via github-glue
+        uses: Chia-Network/actions/github/glue@main
+        with:
+          json_data: '{"test_ref": "${{ env.RELEASE_TAG }}"}'
+          glue_url: "${{ secrets.GLUE_API_URL }}"
+          glue_project: "sync-test/${{ env.RELEASE_TAG }}"
+          glue_path: "start"
 
-      - name: Trigger Workflow
-        run: |
-          curl -s -XPOST -H "Authorization: Bearer ${{ env.JWT_TOKEN }}" --data '{"test_ref": "${{ env.RELEASE_TAG }}"}' ${{ secrets.GLUE_API_URL }}/api/v1/sync-test/$RELEASE_TAG/start
-          curl -s -XPOST -H "Authorization: Bearer ${{ env.JWT_TOKEN }}" --data '{"test_ref": "${{ env.RELEASE_TAG }}"}' ${{ secrets.GLUE_API_URL }}/api/v1/sync-test/$RELEASE_TAG/success/deploy
+      - name: Trigger sync test workflow success via github-glue
+        uses: Chia-Network/actions/github/glue@main
+        with:
+          json_data: '{"test_ref": "${{ env.RELEASE_TAG }}"}'
+          glue_url: "${{ secrets.GLUE_API_URL }}"
+          glue_project: "sync-test/${{ env.RELEASE_TAG }}"
+          glue_path: "success/deploy"
```

Comment:
@cmmarslender I seem to remember there being an endpoint made for this where we can just call start and success in one shot? Or am I misremembering what that was for?

Reply:
Anyway we can either change these over to that (if I remember the usage of that endpoint correctly) now or in a different PR, since currently this PR really only adds http status code handling to the glue service curl requests, and switching endpoints seems more like a second change. But I'm fine with doing it here too.

Reply:
Yep - its the /trigger endpoint. Need to update the config in glue to have no required workflows for the trigger, then swap the endpoint